taca/element/Element_Runs.py

```diff
@@ -428,14 +428,303 @@ def rsync_successful(self):
         else:
             return False
 
-    def aggregate_demux_results(self, demux_results_dirs):
-        # TODO: Correct this based on comments from Chuan
+    # Clear all content under a dir
+    def clear_dir(dir):
+        for filename in os.listdir(dir):
+            file_path = os.path.join(dir, filename)
+            try:
+                if os.path.isfile(file_path) or os.path.islink(file_path):
+                    os.unlink(file_path)
+                elif os.path.isdir(file_path):
+                    shutil.rmtree(file_path)
+            except Exception as e:
+                print(f"Failed to delete {file_path} Reason {e}")
+
+    # Create symlink for a simple demultiplexing dir
+    def symlink_demux_dir(src_dir, dest_dir):
```

Comment:
I can't figure out where this method is called? Is it maybe just planned to be used?

Reply:
Yes, I removed that function. I made it for taking care of the simple cases that only one demux is needed. But then I changed my mind to use the more general function to take care of all cases.

Sorry that I should have deleted it from the scripts!